content/source/docs/enterprise/system-overview/security-model.html.md

```diff
@@ -0,0 +1,100 @@
+
+Due to the extensive capabilities granted to this role, the infrastructure admin role be restricted to a small number of users within your organization.
+
+### Site Admin
+
+[Site admins](../admin/admin-access.html) are responsible for application-level configuration of Terraform Enterprise. Via the admin interface they will be able to manage all users, workspaces and organizations and will have access to all data stored within Terraform Enterprise. Site admins are also responsible for configuring SAML and will be the only users still able to access Terraform Enterprise via username/password once SAML is configured. 
+
+Due to the extensive capabilities granted to this role, the site admin role should be restricted to a small number of users within your organization.
+
+
+## Differences Between Terraform Enterprise and Terraform Cloud Security Models
+
+For the most part, Terraform Enterprise’s threat model can be considered a superset of the [Terraform Cloud security model](../../cloud/architectural-details/security-model.html) and (with the exception of the points listed below) all of the content on that page applies to Terraform Enterprise. Terraform Enterprise’s threat model differs from that of Terraform Clouds in these respects:
+
+### TFE Requires You to Manage and Secure the Underlying Network and Infrastructure
+
+TFE infrastructure admins are required to manage all aspects of the underlying infrastructure including initial provisioning, secure configuration, access control, configuring network ACLs, and OS-level software updates. TFE can not ensure the security of your data if the underlying infrastructure is compromised.
+
+### You are Responsible for Updating Your TFE Deployment
+
+Security fixes are released along with application features and bug fixes via TFE’s standard monthly release cycle. TFE infrastructure admins are responsible for applying updates.
+
+### You are Responsible for TFE’s Availability, Backups, and Disaster Recovery
+
+TFE infrastructure admins are responsible for all aspects of TFE’s reliability and availability. Refer to TFE documentation on [monitoring](../admin/monitoring.html), [backups and restores](../admin/backup-restore.html), and [high availability mode (active/active)](../admin/active-active.html) for more guidance on this topic.
+
+### TFE Isolates Terraform Operations via Docker Containers, not a separate environment
+
+Unlike Terraform Cloud, TFE executes all Terraform operations in Docker containers on the TFE host. The containers are assigned to an isolated Docker network to prevent them from communicating with TFE’s backend services, but may have access to resources available on hosts accessible from the your TFE instance. 
```

Comment:
```suggestion
Unlike Terraform Cloud, Terraform Enterprise executes all Terraform operations in Docker containers on the Terraform Enterprise host. The containers are assigned to an isolated Docker network to prevent them from communicating with Terraform Enterprise backend services, but may have access to resources available on hosts accessible from the your Terraform Enterprise instance. 
```